cmd/list_bundles.go

```diff
@@ -0,0 +1,58 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/opdev/opcap/internal/bundle"
+
+	"github.com/spf13/cobra"
+)
+
+var listBundlesFlags struct {
+	bundlesDir  string
+	bundlesRepo string
+}
+
+func listBundlesCmd() *cobra.Command {
+	cmd := cobra.Command{
+		Use:   "bundles",
+		Short: "List all bundles and versions",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			err := listBundles(cmd.Context(), cmd.OutOrStdout())
+			if err != nil {
+				return err
+			}
+
+			return nil
+		},
+	}
+
+	flags := cmd.Flags()
+
+	flags.StringVar(&listBundlesFlags.bundlesDir, "bundles-path", "internal/bundle/.test/certified-operators",
+		"specifies the source directory with bundles")
```

Comment:
I'm not understanding the utility of this flag. Here, it's stated that it is the source directory for bundles. To me, that means that it is expected to already contain the bundles. However, it is not expected to already contain the repo. Is there a reason that the user would need this to be a specific directory? Does it need to persist beyond the execution of the program?

Reply:
You are right here as well. It's changed. Now we have two mutually exclusive flags: --from-dir --from-repo which allows the user to use a dir with bundles or a different repo from certified operators.